msvc/sdk-build.props

```diff
@@ -29,6 +29,7 @@
     <ApplicationTypeRevision Condition="'$(ApplicationTypeRevision)'==''">10.0</ApplicationTypeRevision>
     <UseStarboardSourceSdk Condition="'$(UseStarboardSourceSdk)'==''">true</UseStarboardSourceSdk>
     <IslandwoodDRT Condition="'$(IslandwoodDRT)'==''">false</IslandwoodDRT>
+    <StarboardLinkWholeArchive Condition="'$(StarboardLinkWholeArchive)'==''">true</StarboardLinkWholeArchive>
```

Comment:
We should definitely evaluate this, and possibly have vsimporter detect it and turn it on.

For now, it's too large a hammer to drop on 3rd-party consumers.